src/test/java/de/caritas/cob/userservice/api/admin/service/agency/RemoveConsultantFromRocketChatServiceTest.java

```diff
@@ -22,20 +22,20 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
-public class RemoveConsultantFromRocketChatServiceTest {
+class RemoveConsultantFromRocketChatServiceTest {
 
   @InjectMocks private RemoveConsultantFromRocketChatService removeConsultantFromRocketChatService;
 
   @Mock private RocketChatFacade rocketChatFacade;
 
   @Mock private ConsultantRepository consultantRepository;
 
-  @Mock private KeycloakService keycloakService;
+  @Mock private IdentityClient identityClient;
```

Comment:
this is from the Upgrade from Tomasz, right?

Reply:
Yes, that was a result of the merge from Tomasz changes.